src/main/java/redis/clients/jedis/JedisPubSub.java

```diff
@@ -15,11 +15,9 @@
 import redis.clients.jedis.exceptions.JedisException;
 import redis.clients.util.SafeEncoder;
 
-public abstract class JedisPubSub {
+public abstract class JedisPubSub extends BinaryJedisPubSub {
```

Comment:
@cpeeyush sure, I'm just waiting for @HeartSaVioR last confirmation about moving forward this way.